meta/open_standup.md

```diff
@@ -0,0 +1,64 @@
+# Dev Team Standup at Artsy
+
+We've been doing engineering standups for as long as there has been an engineering team. The formats have changed over
+time as the size and scope of Artsy has changed. We're now on our fourth iteration of this process.
+
+### Before a standup
+
+* All team leads are requested for updates [via Geekbot](https://geekbot.io) which appears in our engineer team slack.
+* Someone `@here`s the dev channel with a link and where the meeting is in the NYC office
+
+### During standup
+
+* We start off with props, as a nod towards [People are Paramount][pplp]
+* Then we get an update from the CTO
+* Each team lead gives a summary of what their teams have been up to
+* We leave a spot for potential cross-team issues
+* We have a spot to talk about things people are proud of
+* We then either announce [the Lunch & Learn][ll], or try find one for the week
+
+During the standup, someone writes up these notes and passes them out on to slack after. Everyone leaves links to things
```

Comment:
Welllll, kind of. I usually listen for PRs or issues and find them afterwards myself, though sometimes people do leave them in the Slack. Worth mentioning?